modules/imap/scanner.go

```diff
@@ -148,6 +149,21 @@ func getIMAPError(response string) error {
 	return fmt.Errorf("error: %s", response)
 }
 
+// Check the contents of the IMAP banner and return a relevant ScanStatus
+func VerifyIMAPContents(banner string) zgrab2.ScanStatus {
+	switch {
+	case strings.HasPrefix(banner, "* OK"),
```

Comment:
Did some searching on censys.io, and some imap servers actually return `OKAY` as well: https://censys.io/ipv4/198.133.159.137/table#143

We should also match on a `strings.Contains(banner, "IMAP)`?
https://censys.io/ipv4/155.230.11.8/table#143